verde/base/base_classes.py

```diff
@@ -641,12 +644,14 @@ def profile(
         columns.extend(zip(data_names, data))
         return pd.DataFrame(dict(columns), columns=[c[0] for c in columns])
 
-    def _get_dims(self, dims):
+    def _get_dims(self, dims, projection):
```

Comment:
```suggestion
    def _get_dims(self, dims, projection=None):
```

This way classes not passing projection will not break from this change.

Reply:
You're right. I'll also change every call of the method so they can pass the projection as a keyword argument (just in case).